cmd/security-agent/main_windows.go

```diff
@@ -108,9 +107,9 @@ func (s *service) Run(svcctx context.Context) error {
 		orchestratorForwarderImpl.Module(),
 		fx.Supply(orchestratorForwarderImpl.NewDisabledParams()),
 		fx.Provide(func() demultiplexerimpl.Params {
-			opts := aggregator.DefaultAgentDemultiplexerOptions()
-			opts.UseEventPlatformForwarder = false
-			return demultiplexerimpl.Params{Options: opts}
+			params := demultiplexerimpl.NewDefaultParams()
+			params.UseEventPlatformForwarder = false
+			return params
```

Comment:
@ogaca-dd, what are your thoughts of using the options pattern to configure the demultiplexer? https://dev.to/kittipat1413/understanding-the-options-pattern-in-go-390c

Is not a blocker, just want to hear your thoughts 😄

Reply:
The parameters are just boolean. I tend to use the options pattern when there are complex options (For exemple when there is a dependencies between two fields (If `A` is `true` then `B` most be `true`).
In this PR, I tried to limit changes but feel free to implement options pattern if you think it is useful once this PR is merged.